src/components/Error/__tests__/Wrapper.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { create } from 'react-test-renderer';
+import 'jest-styled-components';
```

Comment:
I think that there is no point to import that `jest-styled-components` since we already do that in `src/setupTests.js`.

Reply:
Yeah, I was testing something and left it here. Good spot.